src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -1025,7 +1025,7 @@ $(document).ready(() => {
 
             // disable button to prevent user from clicking submission button again
             const $submissionButton = $('#response_submit_button');
-            addLoadingIndicator($submissionButton, '');
+            addLoadingIndicator($submissionButton, 'Submitting ');
```

Comment:
I intentionally wanted a trailing space so that when clicked the button will show `Submitting ...`, where the `...` is loaded from a GIF. But when I ran this code in my PC I couldn't see the `...` which was supposed to come from the GIF. The same is the case with a few other buttons where the `...` does not appear. So I planned to create a separate issue for it once this is approved. I approved the changes because they are correct as far as the cope of the PR is concerned.